src/Test.FunctionalTests.BinSkim.Driver/BaselineTestsData/Expected/Native_x64_VS2013_Default.dll.sarif

```diff
@@ -638,10 +662,10 @@
                   "text": "'{0}' was not evaluated for check '{1}' as the analysis is not relevant based on observed metadata: {2}."
                 }
               },
-              "shortDescription": {
-                "text": "Binaries should be marked as high entropy Address Space Layout Randomization (ASLR) compatible."
-              },
               "name": "EnableHighEntropyVirtualAddresses",
+              "defaultConfiguration": {

```

Comment:
>defaultConfiguration [](start = 15, length = 20)

this looks bad! this is not a legal value to set this! did the new SARIF SDK introduce this change?